src/test/java/BaseTest.java

```diff
@@ -1,18 +1,61 @@
 import io.github.bonigarcia.wdm.WebDriverManager;
+import org.openqa.selenium.By;
 import org.openqa.selenium.WebDriver;
 import org.openqa.selenium.WebElement;
 import org.openqa.selenium.chrome.ChromeDriver;
-import org.openqa.selenium.firefox.FirefoxDriver;
-import org.openqa.selenium.safari.SafariDriver;
+import org.testng.annotations.AfterMethod;
 import org.testng.annotations.BeforeMethod;
 import org.testng.annotations.BeforeSuite;
-
-
+import org.testng.annotations.Parameters;
+import java.time.Duration;
 public class BaseTest {
-
+    By emailFieldLocator = By.cssSelector("[type = 'email']");
+    By passwordFieldLocator = By.cssSelector("[type = 'password']");
+    By submitButtonLocator = By.cssSelector("[type = 'submit']");
+   By deletePlaylistBtnLocator = By.xpath("//button[@class='del btn-delete-playlist']");
+    By deleteNotifyLocator=By.cssSelector("div.success.show");
+    public static WebDriver driver = null;
+    public static String url = null;
     @BeforeSuite
     static void setupClass() {
         WebDriverManager.chromedriver().setup();
     }
-
-}
+    @BeforeMethod
+    @Parameters({"baseURL"})
+    public void setUpBrowser(String baseURL) {
+        driver = new ChromeDriver();
```

Comment:
After the Chrome 111 update, you can no longer kick off a chromedriver instance unless you add an additional chrome option:
"--remote-allow-origins=*"

 ChromeOptions options = new ChromeOptions();
        options.addArguments("--disable-notifications","--remote-allow-origins=*", "--incognito","--start-maximized");
        driver = new ChromeDriver(options);


Below are the list of available and most commonly used arguments for ChromeOptions class

**start-maximized:** Opens Chrome in maximize mode
**incognito:** Opens Chrome in incognito mode
**headless:** Opens Chrome in headless mode
**disable-extensions:** Disables existing extensions on Chrome browser
**disable-popup-blocking:** Disables pop-ups displayed on Chrome browser
**make-default-browser:** Makes Chrome default browser
**version:** Prints chrome browser version
**disable-infobars:** Prevents Chrome from displaying the notification ‘Chrome is being controlled by automated software

![image](https://user-images.githubusercontent.com/104761482/226286114-c5c29f24-fd2b-480b-b9ea-eaea2510933c.png)